src/content.js

```diff
@@ -14,8 +14,7 @@ const templates = [
   'views',
   'mocks',
   'mocks/steps',
-  'steps',
-  'node_modules/reform-pattern-library/app/views/macros'
```

Comment:
for curiosity, what were these macros for and why have they been removed?

Reply:
they are from old gov uk design template, no longer needed